packages/runtime/datastore-definitions/src/serializable.ts

```diff
@@ -23,4 +23,4 @@ import { Jsonable } from "./jsonable";
  * function serialize<T>(value: Serializable<T>) { ... }
  * ```
  */
-export type Serializable<T = any> = Jsonable<T, IFluidHandle>;
+export type Serializable<T> = Jsonable<T, IFluidHandle>;
```

Comment:
I'm not sure I have a strong opinion here, but would it be convenient to default T to `unknown` here?